--- src/CMakeLists.txt.link	2010-09-29 08:49:34.000000000 +0200
+++ src/CMakeLists.txt	2010-09-29 08:50:12.000000000 +0200
@@ -208,7 +208,7 @@
   )
 endif(KLF_LIBKLFAPP_STATIC)
 add_dependencies(klfapp klftools klfbackend)
-target_link_libraries(klfapp ${QT_LIBRARIES} klfbackend klftools)
+target_link_libraries(klfapp ${QT_LIBRARIES} klfbackend klftools X11)
 set_target_properties(klfapp PROPERTIES
 	#	DEBUG_POSTFIX "${KLF_DEBUG_POSTFIX}"
 	PUBLIC_HEADER "${klfapp_INSTHEADERS}"
